casper/src/test/scala/coop/rchain/casper/api/CreateBlockAPITest.scala

```diff
@@ -89,6 +89,7 @@ private class SleepingMultiParentCasperImpl[F[_]: Monad: Time](underlying: Multi
   def lastFinalizedBlock: F[BlockMessage]          = underlying.lastFinalizedBlock
   def storageContents(hash: ByteString): F[String] = underlying.storageContents(hash)
   def getRuntimeManager: F[Option[RuntimeManager]] = underlying.getRuntimeManager
+  def fetchDependencies: F[Unit]                   = ().pure[F]
```

Comment:
should be `underlying.fetchDependencies`

Reply:
Ah yes. Nice catch